projects/rocprofiler-systems/docker/Dockerfile.opensuse.ci

```
@@ -29,7 +29,7 @@ RUN zypper --non-interactive update -y && \
     zypper --non-interactive install -y -t pattern devel_basis && \
     zypper --non-interactive install -y binutils-gold chrpath cmake curl dpkg-devel \
         gcc-c++ git iproute2 libnuma-devel ninja openmpi3-devel papi-devel python3-devel \
-        python3-pip rpm-build sqlite3-devel vim wget && \
+        python3-pip rpm-build sqlite3-devel vim wget nlohmann_json-devel && \
```

```suggestion
        gcc-c++ git iproute2 libnuma-devel nlohmann_json-devel ninja openmpi3-devel \
        papi-devel python3-devel python3-pip rpm-build sqlite3-devel vim wget && \
```
Sorting list alphabetically.